store/driver/txn/error.go

```diff
@@ -225,7 +239,8 @@ func prettyLockNotFoundKey(rawRetry string) string {
 	if err != nil {
 		return ""
 	}
-	var buf bytes.Buffer
-	prettyWriteKey(&buf, key)
-	return buf.String()
+	var buf1 bytes.Buffer
```

Comment:
We can add a test case for this scenario if possible.

Reply:
I didn't change anything about LockNotFound error messages. So I guess `TestLockNotFoundPrint` is enough.